code/modules/mob/living/carbon/human/species_types/mothmen.dm

```diff
@@ -64,9 +64,12 @@
 	features["moth_markings"] = pick(GLOB.moth_markings_list)
 	return features
 
-/datum/species/moth/get_scream_sound(mob/living/carbon/human/human)
+/datum/species/moth/get_scream_sound(mob/living/carbon/human)
 	return 'sound/voice/moth/scream_moth.ogg'
```

Comment:
```suggestion
	if(!istype(user))
		return
	return 'sound/voice/moth/scream_moth.ogg'
```

Reply:
jacquerel wanted a proc on the species so it was done like this, maybe I misunderstood?